elasticsearch/docker/templates/Dockerfile.j2

```diff
@@ -179,7 +179,8 @@ RUN mkdir /usr/share/elasticsearch && \
     chmod 0775 /usr/share/elasticsearch && \
     chgrp 0 /usr/share/elasticsearch
 
-RUN mkdir /usr/share/supervisor
+RUN mkdir -p /usr/share/supervisor/performance_analyzer/ && \
+    chown 1000 /usr/share/supervisor/performance_analyzer
```

Comment:
This means that only 1000 uid has permission. It will fail if we start with another uid in docker command, right?

Reply:
The dockerfile is created with only elasticsearch/root user in mind anyway.
Unless the user change the Dockerfile with another user added, he will fail to start the container no matter what.
As the user he uses must exist on the container image beforehands.

![image](https://user-images.githubusercontent.com/65193828/87366793-8b619700-c547-11ea-88aa-cc5d552ceed9.png)